packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -822,6 +828,21 @@ class CupertinoTextField extends StatefulWidget {
     properties.add(DiagnosticsProperty<bool>('scribbleEnabled', scribbleEnabled, defaultValue: true));
     properties.add(DiagnosticsProperty<bool>('enableIMEPersonalizedLearning', enableIMEPersonalizedLearning, defaultValue: true));
   }
+
+  static Widget? _iosLoupeBuilder(
+    BuildContext context,
+    LoupeController controller,
+    ValueNotifier<LoupeSelectionOverlayInfoBearer> loupeSelectionOverlayInfoBearer
+  ) {
+    if (defaultTargetPlatform == TargetPlatform.iOS) {
```

Comment:
I did think about using `switch`. I don't think there's a huge difference either way, though, readability or otherwise. I opted for `if` because it's less code, but I'll `switch` (ha) if you think it reads better.